build.gradle

```diff
@@ -45,7 +45,7 @@ repositories {
 dependencies {
     compile gradleApi()
     compile 'org.codehaus.groovy:groovy-all:2.4.4'
-    compile('org.codehaus.groovy.modules.http-builder:http-builder:0.7.2') {
+    compile('org.codehaus.groovy.modules.http-builder:http-builder:0.7.1') {
```

Comment:
This version is available for download (I think Bintray), so let's leave it or upgrade to the latest